modnet/ext_data.py

```diff
@@ -38,10 +39,19 @@ class Usage(Enum):
             "Pickled dataframe containing the Normalized Mutual Information (NMI) between matminer features "
             "computed on the Materials Project."
         ),
-        filename="Features_cross",
+        filename="features_cross",
         md5="b83e0bd43f71ec53c4d69ee0764acfbe",
         usage=Usage["cross_nmi"],
     ),
+    "MP_210321": Dataset(
+        url="https://figshare.com/ndownloader/files/31368412",
+        description=(
+            "A pickled dataframe containing featurized materials from the Materials as of 21 March 2021."
```

Comment:
```suggestion
            "A pickled dataframe containing featurized materials from the Materials Project as of 21 March 2021."
```

Also which features? All of matminer?